lib/tokenValidator.js

```diff
@@ -1,38 +1,45 @@
+"use strict";
+
 // This file borrowed from api.webmaker.org
 
-var Hoek = require('hoek');
-var boom = require('boom');
-var request = require('request');
-var connString = process.env.ID_SERVER_CONNECTION_STRING;
+const Hoek = require(`hoek`);
+const Boom = require(`boom`);
+const request = require(`request`);
+
+const connectionString = process.env.ID_SERVER_CONNECTION_STRING;
 
 Hoek.assert(
-  connString,
-  'You must provide a connection string to a Webmaker Identity Provider (ID_SERVER_CONNECTION_STRING)'
+  connectionString,
+  `You must provide a connection string to an Identity Provider (ID_SERVER_CONNECTION_STRING)`
 );
 
-var req = request.defaults({
-  baseUrl: connString,
-  uri: '/user',
-  method: 'get',
+const requestor = request.defaults({
+  baseUrl: connectionString,
+  uri: `/user`,
+  method: `get`,
   json: true
 });
 
 module.exports = function tokenValidator(token, callback) {
-  req({
+  requestor({
     headers: {
-      authorization: 'token ' + token
+      authorization: `token ${token}`
     }
-  }, function(err, resp, body) {
-    if ( err ) {
-      return callback(err);
+  }, function(error, response, body) {
+    if (error) {
+      return callback(error);
     }
 
-    if ( resp.statusCode !== 200 ) {
-      if ( resp.statusCode === 401 ) {
+    if (response.statusCode !== 200) {
+      if (response.statusCode === 401) {
         return callback(null, false);
```

Comment:
not part of your rewrite but: this feels like it should have just been `callback()` =)